fabric-ca-client/test/FabricCAClient.js

```diff
@@ -1199,33 +1208,9 @@ describe('FabricCAClient', () => {
 			revert = FabricCAClientRewire.__set__('FabricCAClient.prototype.http', requestStub);
 
 			const client = new FabricCAClientRewire(connect_opts, cryptoPrimitives);
-			await client.enroll('enough', 'parameters', 'for_this_function_call').should.be.rejectedWith(/Could not parse enrollment response/);
+			await client.enroll('enough', 'parameters', 'for_this_function_call').should.be.rejectedWith(/Could not parse enroll response/);
 		});
 
-		it('should reject on end point error', async () => {
```

Comment:
What is the reason for removing this test? In general I want to see all the existing tests continue to pass on a refactor to ensure it does not regress behaviour

Reply:
@bestbeforetoday 
Hi Mark, 
This test for a corner case which would only happens in rewire test instead of real case. 
As far as I know. Please clarify if I am wrong. 
This test assume multiple error handler exist both in or out of response callback, which I see it is redundant.

Reply:
But thx, I find that we could register a error handler on response, thus we could handle this case.